docs/data/trigger-sync.md

```diff
@@ -0,0 +1,270 @@
+---
+title: "Trigger Cloud Sync conditionally"
+linkTitle: "Trigger Sync"
+description: "Configure cloud sync to automatically capture data in the Viam app."
```

Comment:
Should this description mention the conditionality part? Also maybe cloud sync is meant to be capitalized, or else it shouldn't be capitalized in the second sentence of this page.